x/ibc/02-client/exported/exported.go

```diff
@@ -154,12 +161,14 @@ type ClientType byte
 const (
 	Tendermint ClientType = iota + 1 // 1
 	Localhost
+	SoloMachine
```

Comment:
ideally they should be sorted according to their ICS number imo
```suggestion
const (
	SoloMachine ClientType = iota + 1 // 1
	Tendermint
	Localhost
```